.github/workflows/frontend-branch.yaml

```
@@ -80,10 +81,10 @@ jobs:
           n=0
           until [ $n -ge 5 ]
           do
-            # TODO: also push to pr-$GITHUB_ACTOR branch. Can do this as a force-push at the end?
             git push origin "HEAD:pr-$GITHUB_HEAD_REF" && break
             n+=1
-            git pull --rebase origin pr-$GITHUB_HEAD_REF
+            git fetch origin pr-$GITHUB_HEAD_REF
+            git merge origin/master --strategy=recursive --strategy-option=ours
```

Do we need to merge the branch `pr-$GITHUB_HEAD_REF` as well? E.g.:

```suggestion
            git merge origin/master origin/pr-$GITHUB_HEAD_REF --strategy=recursive --strategy-option=ours
```

---

ugh. well spotted.